src/app/page/cabinet/cabinet.component.ts

```diff
@@ -33,17 +34,23 @@ export class CabinetComponent implements OnInit {
   public showAlertcouse = true;
   numbersOfCardsThatNeedRepeating: NumberOfCardsThatNeedRepeatingDTO[] = [];
   public status: string;
+  isAuthorized: boolean;
 
   constructor(private deckService: DeckService,
               private courseService: CourseService,
               private cabinetService: CabinetService,
               private router: Router,
               private userStatusChangeService: UserStatusChangeService,
-              private logger: NGXLogger) {
+              private logger: NGXLogger,
+              private logoutService: LogoutService) {
 
   }
 
   ngOnInit(): void {
+    this.isAuthorized = this.logoutService.isAuthorized();
```

Comment:
You don't need the private field isAuthorized as the value is used only once after initialization and not reused.

Reply:
So you recomend to change 
 `if (!this.isAuthorized) `
to 
`if(!this.logoutService.isAuthorized();)`
?

Reply:
Yes, but you can disagree.